Automattic/woocommerce-payments
# KYC: New Onboarding Flow

Fixes #3792  

#### Changes proposed in this Pull Request









#### Testing instructions





* Check out 1773-gh-Automattic/woocommerce-payments-server.
* Go to `http://localhost:8082/wp-admin/admin.php?page=wc-admin&path=%2Fpayments%2Fonboarding`
* Ensure the country value is pre-populated with the WC Admin store currency (if the country is a WCPay supported country).
* Wait for the `<LoadableBlock>` to disappear and then try changing the country, business type, and structure values. Ensure the selects appear/disappear depending on whether the business type selected has structures.
* If all values are selected (either country & type, or country & type & structure), then the task should be marked as completed.

Known issues:

- Currently, the custom OnboardingSelectControl throws an error when a value is selected.
- Need to add placeholder "Please Select" to the selects.
- Need to provide strings for the description of the business type.
 

-------------------


**Post merge**

TODO:
- [ ] Remove the onboarding link from the WC Payments sidebar.
